src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_client_factory.py

```diff
@@ -0,0 +1,14 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.mgmt.batch import BatchManagementClient
+
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+
+import azure.cli.core._logging as _logging
+logger = _logging.get_az_logger(__name__)
```

Comment:
It is good practice to separate the section of import statement from main body of the module.